node_modules/src/web/entrypoint.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
```

Comment:
Why this is required for bicep update? This was already handled as part of `app settings/command line` and run during the actual deployment? @wbreza @weikanglim

Reply:
Looks like git was overzealous about including those files. They're not part of this change.